node-manager/src/nodemanager.rs

```diff
@@ -98,6 +119,59 @@ impl NodeManager {
         self.wallet.sync().await.expect("Wallet failed to sync")
     }
 
+    #[wasm_bindgen]
+    pub async fn new_node(&self) -> String {
```

Comment:
I think this function would make more sense to return `PublicKey` and have a separate function with `#[wasm_bindgen]` that returns calls that and converts it to a string. We will probably end up using this mostly internally so having the `PublicKey` type will be more useful.

Reply:
made a function `create_new_node_from_node_manager` for this